stores/sql/consts.go

```diff
@@ -6,17 +6,34 @@ import (
 	"go.sia.tech/renterd/api"
 )
 
-type ContractState uint8
+type ContractStateEnum uint8
```

Comment:
Let's please not introduce conventions of the dark times ^^

Maybe you can move this to `/stores/sql/types.go` and treat it like the other types that only exist for persistence.
e.g. `type ContractState api.ContractState` and then implement the marshalling.

This type here can in theory remain unchanged for now. Once we stop using it (since we migrated all db code away from gorm), we can get rid of it.

Reply:
I had a naming conflict. I'm not 100% sure what you eventually suggested in your comment, it looks like you gave a suggestion but then changed your mind slightly saying it might be fine to leave this as is for now. I renamed it to `GetContractState` and named this back. I really don't like being inconsistent with `GetX` for just `X` 😛